src/base/bittorrent/session.cpp

```diff
@@ -2432,30 +2432,37 @@ void Session::removeTorrentsQueue() const
     m_resumeDataStorage->storeQueue({});
 }
 
-void Session::setDefaultSavePath(QString path)
+void Session::setSavePath(QString path)
```

Comment:
This parameter can be const ref now. The same to `setDownloadPath()`

Reply:
> This parameter can be const ref now. The same to `setDownloadPath()`

Done.